src/erlcloud_aws.erl

```diff
@@ -124,7 +124,15 @@ aws_request4_no_update(Method, Protocol, Host, Port, Path, Params, Service, #aws
     Query = erlcloud_http:make_query_string(Params),
     Region = aws_region_from_host(Host),
 
-    SignedHeaders = sign_v4(Method, Path, Config, [{"host", Host}], list_to_binary(Query), Region, Service, []),
+    SignedHeaders = case Method of
+                        post ->
+                            sign_v4(Method, Path, Config,
+                                    [{"host", Host}], list_to_binary(Query),
+                                    Region, Service, Params);
```

Comment:
Looks like here we should leave ```Params``` as [] because query params are passed inside request body and not as a part of url. Otherwise we pass them twice.